src/layman/layer/empty_bbox_test.py

```diff
@@ -0,0 +1,41 @@
+import pytest
+from test import process, process_client
+
+workspace = 'test_empty_bbox_workspace'
+
+
+@pytest.fixture(scope="module")
+def ensure_layman():
+    layman_process = process.start_layman()
+    yield
+    process.stop_process(layman_process)
+
+
+@pytest.mark.usefixtures('ensure_layman')
+def test_empty_shapefile():
+    layername = 'empty'
+
+    process_client.publish_layer(workspace, layername, file_paths=[
+        'sample/layman.layer/empty.dbf',
+        'sample/layman.layer/empty.prj',
+        'sample/layman.layer/empty.qpj',
+        'sample/layman.layer/empty.shp',
+        'sample/layman.layer/empty.shx',
+    ])
+
+    process_client.delete_layer(workspace, layername)
```

Comment:
Testuje tohle to, v čem byl hlášený problém? Pochopil jsem to tak, že vytvoření prázdné vrstvy se povedlo a selhal až následný patch.

Reply:
Vytvoření prázdné vrstvy se nepovedlo, selhalo generování thumbnailu.

Větší problém ale byl, že prázný BBOX nevyhovoval některým WMS klientům, aspoň tak jsem to pochopil.